src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+// helper fn
+function isMan(person) {
+  return person.sex === 'm';
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredTab = people
+    .filter(isMan)
+    .filter(haveDiedInCentury);
+
+  function haveDiedInCentury(person) {
+    return (century) ? (Math.ceil(person.died / 100)) === century : true;
+  }
+
+  function sumAllAges(prev, next) {
```

Comment:
The `sumAllAges` function should be moved outside of `calculateMenAverageAge` to keep the main function cleaner, and to avoid redefining it on each function call. 😇